bondoc-jd-inventory/InventorySystem/forms/insertEmployee.php

```diff
@@ -0,0 +1,51 @@
+<?php
+$email = $_POST['email'];
+$fname = $_POST['fname'];
+$lname = $_POST['lname'];
+$age = $_POST['age'];
+
+if (!empty($email) || !empty($fname) || !empty($lname) || !empty(age)){
+    $host = "localhost";
+    $dbUsername = "root";
+    $dbPassword = "";
+    $dbname = "user_db";
+
+    $conn = new mysqli($host, $dbUsername, $dbPassword, $dbname);
+
+    if (mysqli_connect_error()){
+        die('Connect Error('. mysqli_connect_errno().')'. mysqli_connect_error());
+    }else{
+        $SELECT = "SELECT email From user_employee Where email = ? Limit 1";
+        $INSERT = "INSERT Into user_employee (email, firstname, lastname, age) values(?, ?, ?, ?)";
+
+
+        $stmt = $conn->prepare($SELECT);
+        $stmt->bind_param("s", $email);
+        $stmt->execute();
+        $stmt->bind_result($email);
+        $stmt->store_result();
+        $rnum = $stmt->num_rows;
+
+        if ($rnum==0){
+            $stmt->close();
+
+            $stmt = $conn->prepare($INSERT);
+            $stmt->bind_param("sssi", $email, $fname, $lname, $age);
+            $stmt->execute();
+            echo "<script>alert('Success');
+            window.location.href='http://localhost/Code/adminPage.php';</script>";
```

Comment:
`http://localhost/` ano mangyayari kapag ibang pc un nag access?   magredirect pa din ba or hindi?

Reply:
Hindi pa sya pwede ma access sa iba sir nasa localhost pa po. I'll try to do the port today sir ill update nalang kung naayos kona po.